03.asynchronous/promise.js

```diff
@@ -0,0 +1,43 @@
+import {
+  openDatabaseConnection,
+  closeDatabaseConnection,
+  runDatabaseQuery,
+  getDatabaseData,
+} from "./db_utils.js";
+
+let newDb = undefined;
+
+openDatabaseConnection(":memory:")
+  .then((db) => {
+    newDb = db;
```

Comment:
`newDb` にするのは 11 行目の引数のほうです。

```suggestion
  .then((newDb) => {
    db = newDb;
```

Reply:
すみません、勘違いしていました。
修正しました。